src/webextension/list/popup/components/item-list-panel.css

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.filter-bar {
+  padding: 12px;
+}
+
+.search-header {
+  height: auto;
```

Comment:
I'm not a big fan of this because it means that the header on the list panel and the header on the details panel are different heights. It'd be great if we could get them to be the same height. This is another thing for @changecourse to think about, but I've been considering how we'd handle transitions between the list and details view of the doorhanger.

From my experience with mobile UI design on Firefox OS, the doorhanger looks remarkably similar (the various Panel components I designed are very much meant to mirror Firefox OS and other mobile layouts). With that in mind, I'd *like* it if the headers were the same height (*and* the same color) so that it appears that these are actually the same element. For transitions, we could then, say, immediately replace the header (so it appears that just the text was updated) and then slide in the detail panel from the right (using the Photon animation curve, of course). We'd do the reverse animation when hitting the Back arrow.

This does, however, necessitate some thought about what the "generic" header/body/footer styles should look like for Panels. @changecourse, would you like to set up a meeting sometime this week to discuss this?